packages/block-editor/src/components/block-list/moving-animation.js

```diff
@@ -92,7 +91,6 @@ function useMovingAnimation(
 
 			return;
 		}
-		ref.current.style.transform = 'none';
```

Comment:
I don't understand why this is here, and there's no clarifying comment. Normally the code below is responsible for setting the transform style, which is either translate3d or not defined. Things seems to work fine without these lines. Cc @youknowriad

Reply:
I guess it's because of the `getAbsolutePosition` call right after. If you keep the currently applied transform, the destination position will be wrong (probably useful in the case of multiple clicks on the movers quickly, maybe other situations can lead to a transform applied too)

Reply:
Oh, I read that as getting the positioned element. :)